services/motion/builtin/move_request.go

```diff
@@ -67,6 +66,7 @@ type moveRequest struct {
 	obstacleDetectors map[vision.Service][]resource.Name
 	replanCostFactor  float64
 	fsService         framesystem.Service
+	absoluteFS        referenceframe.FrameSystem
```

Comment:
can we just name this `fs` or `frameSystem`?

Reply:
I would like to have a descriptive name for the field name, what are your thoughts on `checkPlanFS`. 
I want to communicate through this name that we have the following elements in the FS:
```
checkPlanFS := localizationFrame -- planningFrame -- other_children
```
opposed to 
```
regular_FS_topology_we've_used_until_now: := planningFrame -- other_children
```

Reply:
How about localizingFS then? I think the word absolute can be a little confusing

Reply:
sounds good to me!